src/applications/vaos/tests/new-appointment/components/VAFacilityPage/VAFacilityPageV2.multi.unit.spec.jsx

```diff
@@ -118,8 +118,8 @@ const facilities = vhaIds.map((id, index) => ({
     ...getVAFacilityMock().attributes,
     uniqueId: id.replace('vha_', ''),
     name: `Fake facility name ${index + 1}`,
-    lat: 40.8596747, // Clifton, NJ
-    long: -74.1927881,
+    lat: Math.random() * 90,
+    long: Math.random() * 180,
```

Comment:
Needed random coordinates to verify sort.

Reply:
I'm sort of surprised this doesn't break other tests